packages/sdk/src/utils.ts

```diff
@@ -0,0 +1,2 @@
+export const tunaMirrorURI = 'https://pypi.tuna.tsinghua.edu.cn/simple';
+export const JobStatus = [ 'creating', 'running', 'success', 'fail' ];
```

Comment:
Ditto, could we define these constants at core?

Reply:
The package SDK is not complete now, how about doing this after all apis have been done?

Reply:
Ok, so we need to add a FIXME comment to document this.